spec/docx/smoke/api_stroke_spec.rb

```diff
@@ -6,4 +6,17 @@
     docx = builder.build_and_parse('js/docx/smoke/api_stroke/get_class_type.js')
     expect(docx.elements[1].nonempty_runs.first.text).to eq('Class Type = stroke')
   end
+
+  it 'ApiStroke | ToJSON method' do
+    docx = builder.build_and_parse('js/docx/smoke/api_stroke/to_json.js')
+    expect(docx.elements.first.nonempty_runs.first.alternate_content.office2010_content.graphic.type).to eq(:shape)
+    stroke = JSON.parse(docx.elements[1].nonempty_runs.first.text)['graphic']['spPr']['ln']
+    expect(stroke['w']).to eq(180_000)
+    expect(OoxmlParser::Color.new(stroke['fill']['fill']['color']['color']['rgba']['red'],
+                                  stroke['fill']['fill']['color']['color']['rgba']['green'],
+                                  stroke['fill']['fill']['color']['color']['rgba']['blue']).to_hex.to_sym)
+      .to eq(OoxmlParser::Color.new(51, 51, 51).to_hex.to_sym)
```

Comment:
I think the conversion to hex of the color object is not required, since we are comparing two color objects, not an color object and a string\symbol

Try using `expect(smt).to eql(smt)`
 
```doc
a.eql?(b) # object equivalence - a and b have the same value
```